### PR TITLE
Include a warning in regards using WSL with Windows 10 1803 build

### PR DESCRIPTION
**Pull request by Ralms**
Add a warning at WSL usage in regards to Windows 10 1803 build having issues with CMake.

### CODE REVIEW

File: doc/getting_started/installation_win.rst [hunk 1]
```diff
@@ -347,6 +347,10 @@ built-in functionality to natively run Ubuntu binaries directly on a standard
 command-prompt. This allows you to install the standard Zephyr SDK and build
 for all supported architectures without the need for a Virtual Machine.
 
+.. warning::
+      Windows 10 1803 has an issue that will cause cmake to not work properly.
```
Comment (dbkinder) [inline doc/getting_started/installation_win.rst]:
```suggestion
      Windows 10 version 1803 has an issue that will cause cmake to not work properly
      and is fixed in version 1809 (and later).
```

File: doc/getting_started/installation_win.rst [hunk 2]
```diff
@@ -347,6 +347,10 @@ built-in functionality to natively run Ubuntu binaries directly on a standard
...
+      More information can be found here: `Zephyr Issue 10420`_
```
Comment (dbkinder) [inline doc/getting_started/installation_win.rst]:
```suggestion
      More information can be found in `Zephyr Issue 10420`_
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Since this issue appears fixed in a later Windows release, we should mention that.

**Review by dbkinder — APPROVED** [review]
LGTM, but there are commit message issues that need to be fixed:  https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/24640/5/tests

**Comment by dbkinder** [comment]
Looks like the new GitHub "suggest" comment feature creates a separate commit for each suggestion accepted, and these need to be squashed to work properly within our project workflow (these kind of commits don't have a signed-off).  Sigh.

**Review by tyrantlizard77 — APPROVED** [review]

**Comment by carlescufi** [comment]
@Ralms can you please rebase and fix the commit message?

**Comment by carlescufi** [comment]
@Ralms sorry for the delay, I missed this message.

You basically do:

```
git checkout patch-1
git fetch upstream
git rebase -i upstream/master
<edit commit messages so they comply with the requirements>
```

**Comment by carlescufi** [comment]
@Ralms I've fixed your commit and pushed it, hope that's OK.

**Review by carlescufi — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Comment by Ralms** [comment]
@carlescufi Thank you. 
I was having some issues with the rebase command and than forgot about this pull request, I apologize.

**Pull request merged**